docs/redoc/master/openapi.json

```diff
@@ -9725,6 +9929,134 @@
             "type": "string"
           }
         ]
+      },
+      "DiscoverRequest": {
+        "description": "Use context and a target to find the most similar points, constrained by the context.\n\nWhen using only the context, a special search is performed where pairs of points are used to generate a loss that guides the search towards the zone where most positive examples overlap. This means that the score minimizes the scenario of finding a point closer to a negative than to a positive part of a pair. Since the score of a context relates to loss, the maximum score a point can get is 0.0, and it becomes normal that many points can have 0.0 as score.\n\nUsing only a target is equivalent to regular search, so the score is the distance to the target.\n\nWhen using both context and target, the score behaves a little different: The integer part of the score represents the \"rank\" with respect to the context, while the decimal part of the score relates to the distance to the target.",
```

Comment:
Would it be possible to make this definition shorter, so it would be consistent with all other definitions?

Reply:
Moved the `target` and `context` descriptions to their respective fields 👍